gateway/webhook/webhookTransformer.go

```diff
@@ -36,15 +35,13 @@ type transformerBatchResponseT struct {
 }
 
 func (bt *batchWebhookTransformerT) markResponseFail(sourceType, reason string) transformerResponse {
+	statusCode := response.GetErrorStatusCode(reason)
 	resp := transformerResponse{
 		Err:        response.GetStatus(reason),
-		StatusCode: response.GetErrorStatusCode(reason),
+		StatusCode: statusCode,
 	}
 	bt.stats.failedStat.Count(1)
-	stats.Default.NewTaggedStat("source_transform_failed_jobs", stats.CountType, stats.Tags{
-		"sourceType": sourceType,
-		"statusCode": strconv.Itoa(response.GetErrorStatusCode(reason)),
-	}).Count(1)
+	countWebhookErrors(sourceType, statusCode, 1)
```

Comment:
do we really want to include this here? won't this lead to duplicates?

Reply:
Yes removed from this method